content/docs/guides/partner-intro.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Partner guide
+subtitle: Learn how you can partner with Neon
+enableTableOfContents: true
+isDraft: false
+updatedOn: '2023-11-03T15:15:02.553Z'
+---
+
+Find relevant documentation explaining how partners can integrate with Neon, how to set usage quotas, and links to where you can apply to become a Neon partner.
+
+## Partnering with Neon
+
+Apply for partnership with Neon.
+
+<DetailIconCards>
+
+<a href="https://neon.tech/partners" description="Discover the benefits of partnering with Neon for serverless Postgres" icon="hourglass">Partner Page</a>
+
+<a href="https://neon.tech/partners#partners-apply" description="Request partnership online" icon="setup">Apply</a>
+
+</DetailIconCards>
+
+## Integrate with Neon
+Find details about the different ways you can integrate with Neon.
+
+<DetailIconCards>
+
+<a href="/docs/guides/oauth-integration" description="Integrate with Neon using OAuth" icon="hourglass">OAuth</a>
+
+<a href="/docs/reference/api-reference" description="Integrate using the Neon API" icon="setup">API</a>
+
+<a href="https://neon-experimental.vercel.app/" description="See a sample application using OAuth" icon="hourglass">Sample OAuth</a>
+
+</DetailIconCards>
+
+## Billing
+
+Learn how to set up quotas on key consumption metrics for your projects.
+
+<DetailIconCards>
+
+<a href="/docs/guides/partner-billing" description="Use the Neon API to configure billing quotas for your customers" icon="setup">Set up usage quotas</a>
```

Comment:
The text mentions "billing quotas" and the link says "usage quotas". Should we use one or the other?

Reply:
Changed to: 

Use the Neon API to configure consumption quotas for your customers.

Set up usage quotas

Question: are "consumption" and "usage" synonymous enough it's a benefit to use both terms here, short in the title, longer in description?